inc/class-settings.php

```diff
@@ -283,12 +283,14 @@ public function on_admin_enqueue_scripts() {
 			return;
 		}
 
-		wp_enqueue_style(
-			'ai-logger-admin',
-			AI_LOGGER_URL . 'static/css/admin.css',
-			[],
-			'0.1',
-		);
+		wp_enqueue_style( 'ai-logger-admin', AI_LOGGER_URL . 'static/css/admin.css', [], '0.1' );
+		wp_enqueue_style( 'prism', 'https://cdnjs.cloudflare.com/ajax/libs/prism/1.29.0/themes/prism.min.css', [], '1.29.0' );
```

Comment:
I'd recommend storing the Prism version as a variable (maybe a constant on this class, even) and inserting it as a variable in both the URL here and in the version string. That way you could update to a newer version of Prism simply by bumping the version number.